ledger-core/v2/virtual/execute/execute_isolation_negotiation_test.go

```diff
@@ -178,7 +186,10 @@ func Test_Execute_stepIsolationNegotiation(t *testing.T) {
 					return smachine.StateUpdate{}
 				})
 			} else {
-				execCtx.JumpMock.Set(stepChecker.CheckJumpW(t))
+				execCtx.JumpMock.Set(func(s1 smachine.StateFunc) (s2 smachine.StateUpdate) {
```

Comment:
Try to avoid multiline checking of function equality, extract it to helper pls.

Reply:
Apply same everywherre

Reply:
I'm not sure what do you mean by that
If something goes wrong require will explicitly will say what it has and what is expected

Do you mean wrapper around require?
The check right now is single line
`require.Equal(t, reflectkit.CodeOf(smExecute.stepTakeLock), reflectkit.CodeOf(s1))`

Reply:
yes, something like `testutils.CompareJump(t, smExecute.stepTakeLock)` and 
```
package testutils

func CompareJump(t *testing.T, expected interface{}) func (smachine.StateFunc) smachine.StateUpdate {
    return func(got smachine.StateFunc) smachine.StateUpdate {
        require.Equal(t, reflectkit.CodeOf(expected), reflectkit.CodeOf(got))
    }
}

Reply:
done